### DISCUSSION TITLE
Repo locked by LFS overflow

### DISCUSSION THREAD

**Discussion by ghost** [community/community#42785] [Repositories]
### Select Topic Area

Question

### Body

Hi, my repository is locked because i overflow LFS capacity.

In repo settings i see i need to wait 5 weeks, but i waiting a lot more, like 11 but always locked.

How can i clone my last push to work on, please ?

**Comment by ghost** [comment]
Hi, nobody can help me please ?

Maybe i badly search. If it is, can you please give me correct search words ?

Thanks by advance.

**Comment by CodesdaLu** [comment]
Hey, 
If you choose to include Git LFS objects in archives of your repository, every download of those archives will count towards bandwidth usage for your account. Each account receives 1 GB per month of bandwidth for free, and you can pay for additional usage.

see this link is a documentation github explains everything.

About Git LFS objects in archives  >>> https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-git-lfs-objects-in-archives-of-your-repository

About storage and bandwidth usage >>>> https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage

Gerenciando a cobrança do Git Large File Storage >>  https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage


If not, contact support.


if it helped mark as answer answered
